### PR TITLE
feature(ui-ux): add portfolio 2.0 header and address selection button v2

### PR DESCRIPTION
**Pull request by kyleleow**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- To update portfolio screen header, no bottom border is added for now to not create a separation with content in 2.0
- To add address selection button v2

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

**Comment by chloezxyy** [comment]
Screenshot 2022-07-20 at 12 03 54 PM

Address label emoji's are slightly cut off on android

**Comment by kyleleow** [comment]
It happens on certain combination of text and emoji, but not every time
image

**Review by pierregee — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx [hunk 1]
```diff
@@ -460,6 +446,29 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
           />
         }
       >
+        <ThemedViewV2
```
[the remark this answers is not in the document]
Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx:449]: will be handled in another pr

**Review by thedoublejay — CHANGES_REQUESTED** [review]
Screenshot_20220721-113345_Expo Go

Address Header is overflowing

**Pull request merged**